OpenTyping/MainWindow.xaml.cs

```diff
@@ -94,6 +94,12 @@ public MainWindow()
                 Settings.Default[ProgramLang] = "uz"; // App default language
             }
 
+            if (File.Exists("./errors.log") && InternetServices.IsConnected())
```

Comment:
To increase the speed of App's starting, you can make the error reporting by a asynchronous thread to work independently.